.travis.yml

```diff
@@ -10,8 +10,8 @@ cache:
     - $HOME/.composer/cache
 
 before_install:
-  - composer require "illuminate/http:${ILLUMINATE_VERSION}" --no-update
-  - composer require "illuminate/validation:${ILLUMINATE_VERSION}" --no-update
+  - composer require "illuminate/http:6.11.*" --no-update
```

Comment:
we dont want to lock it to this version so we should probably just delete this before_install